util/points.js

```diff
@@ -143,23 +143,62 @@ async function pointsFromSteamID (steamid, context = epochtal) {
  */
 async function calculatePointsDelta (context = epochtal) {
 
-  const boards = await leaderboard(["list"], context);
+  // Get a clone of all final leaderboards
+  const allBoards = context.data.leaderboard;
```

Comment:
I find this comment here to be a bit misleading as it implies the single line following it is the entire clone, rather than the entire following bit.